.github/workflows/test-bundlers.yml

```diff
@@ -0,0 +1,28 @@
+name: Bundler Tests
+
+on:
+  workflow_call:
+    # Make this a reusable workflow, no value needed
+    # https://docs.github.com/en/actions/using-workflows/reusing-workflows
+
+jobs:
+  bundler_tests:
+    name: Verify Bundlers
+    runs-on: 'ubuntu-latest'
```

Comment:
just wondering - is it total overkill / not necessary to test this across the `node` and `os` matrix like we are doing for the e2e tests and whatnot?

Reply:
I think so - although we need to validate runtime behavior (the component is lazily loaded correctly), I don't think it's pertinent that we test this across all Node versions/OS's.